clients/cascade/src/app/CalibrationPage.vue

```diff
@@ -338,9 +338,9 @@ Last update: 2018-09-06
       projectID()    { return utils.projectID(this) },
       hasData()      { return utils.hasData(this) },
       hasPrograms()  { return utils.hasPrograms(this) },
-      simStart()     { return utils.simStart(this) },
-      simEnd()       { return utils.simEnd(this) },
-      simYears()     { return utils.simYears(this) },
+      simStart()     { return utils.dataStart(this) },
+      simEnd()       { return utils.dataEnd(this) },
+      simYears()     { return utils.dataYears(this) },
```

Comment:
this isn't right...i'm not sure what you're trying to do, but e.g. simEnd should be utils.simEnd, not utils.dataEnd